app/services/testdata/Phase3TestsInvitedStatusGenerator.scala

```diff
@@ -16,18 +16,24 @@
 
 package services.testdata
 
-import repositories._
+import java.util.UUID
+
+import _root_.services.onlinetesting.Phase3TestService
 import config.LaunchpadGatewayConfig
 import config.MicroserviceAppConfig._
+import connectors.testdata.ExchangeObjects.DataGenerationResponse
 import model.ApplicationStatus._
+import model.OnlineTestCommands.OnlineTestApplication
 import model.command.testdata.GeneratorConfig
+import model.persisted.phase3tests.{ LaunchpadTest, Phase3TestGroup }
+import org.joda.time.DateTime
 import play.api.mvc.RequestHeader
-import repositories.onlinetesting.{ Phase1TestRepository, Phase3TestRepository }
-import _root_.services.onlinetesting.Phase3TestService
-import model.OnlineTestCommands.OnlineTestApplication
+import repositories._
+import repositories.onlinetesting.Phase3TestRepository
 import uk.gov.hmrc.play.http.HeaderCarrier
 
 import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
 
 object Phase3TestsInvitedStatusGenerator extends Phase3TestsInvitedStatusGenerator {
   override val previousStatusGenerator = Phase2TestsStartedStatusGenerator
```

Comment:
This probably wants its previous generator to be the latest Phase2 one we can find, which is at the moment Phase2ResultsReceivedStatusGenerator